Easydict/objc/MMKit/Log/MMFileLogFormatter.m

```diff
@@ -41,16 +41,16 @@ - (NSString *)formatLogMessage:(DDLogMessage *)logMessage {
             logLevel = @"❌";
             break;
         case DDLogFlagWarning:
-            logLevel = @"W";
+            logLevel = @"⚠️";
             break;
         case DDLogFlagInfo:
-            logLevel = @"I";
+            logLevel = @"ℹ️";
             break;
         case DDLogFlagDebug:
-            logLevel = @"D";
+            logLevel = @"Debug";
             break;
         default:
-            logLevel = @"V";
+            logLevel = @"Verbose";
```

Comment:
We should keep the formats of `MMFileLogFormatter` and `MMConsoleLogFormatter` consistent. There is some code duplication here, so we can optimize it.

Reply:
I tried but not familiar with OC implementation, could you help on this optimization?

Reply:
ok, I will do this later.